app/src/lib/git/rev-parse.ts

```diff
@@ -52,6 +52,30 @@ export async function getTopLevelWorkingDirectory(
   return Path.resolve(path, relativePath)
 }
 
+/**
+ * Checks if the repository at a path is bare.
```

Comment:
It looks like this code can handle a non-Git repository being at the path, but it errors if the path doesn't exist on disk. Can we capture that here and in the `@param path` value below?

Reply:
@shiftkey 
In regards to these two changes requested, would the following be sufficient wording? If so I will submit it in another commit.

```
/**
 * Checks if the repository at a path is bare.
 *
 * @param path The path to the repository to check. An error will be thrown if the path does not exist on disk.
 *
 * @returns true if the path contains a bare Git repository. Returns false if it is not bare or is not a Git repository.
 */
```

Reply:
@Daniel-McCarthy works for me